internal/database/dbutil/schema.go

```diff
@@ -68,9 +68,12 @@ func BuildIndexDDL(tableName string, indexName string, fields []string, backend
 	qt := QuoteFn(backend)
 	switch backend {
 	case types.BackendCassandra,
-		types.BackendMySQL,
-		types.BackendSQLite:
+		types.BackendMySQL:
 		return fmt.Sprintf("CREATE INDEX %s ON %s (%s)", qt(indexName), qt(tableName), qt(fields...))
```

Comment:
should we also make other database's index unique?

Reply:
I also feel the same way, use a unique name here in oomstore to save one DB by one judgment.

Reply:
done